src/Calculator.macOS/MenuHelper.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using AppKit;
+
+namespace calculator.macOS
+{
+	public static class MenuHelper
+	{
+		internal static NSMenu GetMenu()
+		{
+			var menubar = new NSMenu();
```

Comment:
You should be able to use the MenuBar control, with a style:

https://github.com/unoplatform/uno/blob/b7ccfcae1e4a76044b86e572802049869d9e746e/src/Uno.UI/UI/Xaml/Controls/MenuBar/MenuBar.xaml#L31